releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,581 @@
+    - Encode the values of RawKV API V2 as `user-value` + `ttl` + `ValueMeta` and encode `delete` in `ValueMeta` [#11965](https://github.com/tikv/tikv/issues/11965)
+    - TiKV Coprocessor supports the Projection operator [#12114](https://github.com/tikv/tikv/issues/12114)
+    - Support dynamically modifying `raftstore.raft-max-size-per-msg` [#12017](https://github.com/tikv/tikv/issues/12017)
+    - Support monitoring multi-k8s in Grafana [#12014](https://github.com/tikv/tikv/issues/12014)
+    - Transfer the leadership to CDC observer to reduce latency jitter [#12111](https://github.com/tikv/tikv/issues/12111)
+    - Support dynamically modifying `raftstore.apply_max_batch_size` and `raftstore.store_max_batch_size` [#11982](https://github.com/tikv/tikv/issues/11982)
+    - RawKV V2 returns the latest version upon receiving the `raw_get` or `raw_scan` request [#11965](https://github.com/tikv/tikv/issues/11965)
+    - Support the RCCheckTS consistency reads [#12097](https://github.com/tikv/tikv/issues/12097)
+    - Support dynamically modifying  `storage.scheduler-worker-pool-size`(the thread count of the Scheduler pool) [#12067](https://github.com/tikv/tikv/issues/12067)
+    - Control the use of CPU and bandwidth by using the global foreground flow controller to improve the performance stability of TiKV [#11855](https://github.com/tikv/tikv/issues/11855)
+    - Support dynamically modifying `readpool.unified.max-thread-count` (the thread count of the UnifyReadPool)  [#11781](https://github.com/tikv/tikv/issues/11781)
+    - Use the TiKV internal pipeline to replace the RocksDB pipeline and deprecate the `rocksdb.enable-multibatch-write` parameter [#12059](https://github.com/tikv/tikv/issues/12059)
+
++ PD
+
+    - Support automatically selecting the fastest object for transfer when evicting the leader, which helps speed up the eviction process  [#4229](https://github.com/tikv/pd/issues/4229)
+    - Forbid deleting a voter from a 2-replica Raft group in case that the Region becomes unavailable [#4564](https://github.com/tikv/pd/issues/4564)
+    - Speed up the scheduling of the balance leader [#4652](https://github.com/tikv/pd/issues/4652)
+
++ TiFlash
+
+    - Forbid the logical splitting of TiFlash files (by adjusting the default value of `profiles.default.dt_enable_logical_split` to `false`. See [user document](/tiflash/tiflash-configuration.md#tiflash-configuration-parameters) for details) and improve the space usage efficiency of the TiFlash columnar storage so that the space occupation of a table synchronized to TiFlash is similar to the space occupation of the table in TiKV.
+    - Optimize the cluster management and replica replication mechanism for TiFlash by integrating the previous cluster management module into TiDB, which accelerates replica creation for small tables. [#29924](https://github.com/pingcap/tidb/issues/29924)
```

Comment:
```suggestion
    - Forbid the logical splitting of TiFlash files (by adjusting the default value of `profiles.default.dt_enable_logical_split` to `false`. For details, see [user document](/tiflash/tiflash-configuration.md#tiflash-configuration-parameters).) and improve the space usage efficiency of the TiFlash columnar storage so that the space occupation of a table synchronized to TiFlash is similar to the space occupation of the table in TiKV.
    - Optimize the cluster management and replica replication mechanism for TiFlash by integrating the previous cluster management module into TiDB, which accelerates replica creation for small tables [#29924](https://github.com/pingcap/tidb/issues/29924)
```